docs/2.5. APIs - Query Parameters.md

```diff
@@ -12,60 +12,65 @@ Query APIs SHOULD support pagination of their API resources where the 'paged' tr
+Paging through a collection for a second time, using the same cursors as previously SHOULD NOT result in new data appearing in the response payloads when using 'creation' ordering, provided the requested paging times are less than or equal to the maximum paging time held in the registry. However, note that as the registry is dynamic and resources MAY be updated or deleted a given cursor MAY return fewer results than it did previously. When using 'update' ordering and a query filter, new data MAY appear when cursors are used for a second time.
 
-* When query parameters which perform filtering are used at the same time as paging, the filters must be applied by the implementation before applying paging parameters to the resulting data set.
+When query parameters which perform filtering are used at the same time as paging, the filters MUST be applied by the implementation before applying paging parameters to the resulting data set.
 
-* A server may choose its own default value for paging limit (see Example 1).
+A server MAY choose its own default value for paging limit (see Example 1).
 
-* Where both 'since' and 'until' parameters are specified, the 'since' value takes precedence where a resulting data set is constrained by the server's value of 'limit' (see Example 5).
+Where both 'since' and 'until' parameters are specified, the 'since' value takes precedence where a resulting data set is constrained by the server's value of 'limit' (see Example 5).
 
-* Servers should include 'prev' and 'next' Link headers and may include 'first' and 'last' Link headers in all responses.
+Servers SHOULD include 'prev' and 'next' Link headers and MAY include 'first' and 'last' Link headers in all responses.
 
 ### Examples
 
-The following examples aim to identify how pagination should behave in the presence of a set of registered data. In order to avoid displaying full resource representations, the only data listed here is the 'update' timestamp associated with each registered record. The same procedures can be applied where 'creation' timestamps and the '?paging.order=create' parameter are used instead.
+The following examples show pagination for a set of registered data. In order to avoid displaying full resource representations, the only data listed here is the 'update' timestamp associated with each registered record. The same procedures can be applied where 'creation' timestamps and the '?paging.order=create' parameter are used instead.
 
 Where a paging limit is not specified in a request the server's default is used.
 
 **Sample Data: Registered Node Update Timestamps (Comma-Separated)**
-```
+
+```json
```

Comment:
This example is not JSON (even though it has `[` and `]` to indicate an array).

Given that the suggestion would be to just to go back to three back-ticks, it's impossible to write!!

Or quote the timestamps?
Or accept that it's not valid JSON?
Hmmm...

Reply:
Well at least at the moment the linter doesn't check this sort of thing so lets leave it as pseudo-json for now

Reply:
Agreed.